config/locales/en/claims/pages.yml

```diff
@@ -17,5 +17,108 @@ en:
         provider_guidance_link: Guidance for providers on initial teacher training (ITT)
         trn_guidance_link: Find out what a teacher reference number (TRN) is and how to find or request a TRN
         related_content: Related content
-      feedback:
-        page_title: Feedback
+      accessibility:
+        page_title: Accessibility statement for Claim funding for mentor training
+        guidance: "This service is run by the Becoming a Teacher team at the Department for Education. We want as many people as possible to be able to use the service. For example, that means users should be able to:"
+        accessibility_colours: change colours, contrast levels and fonts
+        accessibility_zoom: zoom in up to 300% without the text spilling off the screen
+        accessibility_keyboard_navigation: navigate most of the service using just a keyboard
+        accessibility_speech_recognition: navigate most of the service using speech recognition software
+        accessibility_screen_reader: listen to most of the content on the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+        text_on_service: We’ve also made the text on the service as simple as possible to understand.
+        ability_net: AbilityNet
+        ability_net_advice_html: "%{link_to} has advice on making your device easier to use if you have a disability."
+        technical_information: Technical information about this service’s accessibility
+        technical_information_guidance: The Becoming a Teacher team (DfE) is committed to making Claim funding for mentors accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+        web_accessibility_guidelines_html: This website is partially compliant with the  %{link_to} AA standard, due to the non-compliances listed below.
+        web_accessibility_guidelines: Web Content Accessibility Guidelines (WCAG) version 2.1
+        service_design_html: The service was designed using the %{link_to}, which conforms to the WCAG 2.1 AA standard. To ensure the service fully meets the WCAG 2.1 AA standard, we are currently arranging an accessibility audit. We will update this page following the audit, and publish details of any accessibility issues identified.
```

Comment:
I'm not entirely sure that locale files are the right tool for these pages. These are static pages of long-form content, rather than typical 'web app' pages made up of buttons and structured strings of UI labels, etc.

I think this could introduce friction when it comes to content changes. For example, removing paragraphs, adding new ones, or restructuring them altogether will lead to lots of cognitive load. Key names could fall out of sync with the content the represent, and the file could easily end up with orphaned keys, etc.

Instead, have you considered using [Rails localised views](https://guides.rubyonrails.org/i18n.html#localized-views) for these content-heavy pages?

From the Rails guide:

> You can make use of this feature, e.g. when working with a large amount of static content, which would be clumsy to put inside YAML or Ruby dictionaries. Bear in mind, though, that any change you would like to do later to the template must be propagated to all of them.

Since we only actually support one locale (English), and have no plans to change that in the future, I think this could be the most pragmatic approach for these content-heavy pages.

What do you think?

Reply:
I've moved the content on the `.html` file